packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.types.ts

```diff
@@ -94,6 +108,57 @@ export interface IScrollablePaneStyles {
   contentContainer: IStyle;
 }
 
+/**
+ * {@docCategory ScrollablePane}
+ */
+export interface IStickyContainerBehavior {
+  /**
+   * If true, it replicates actual element instead of keeping placeholder for the component which is to be sticky'ed.
+   * Calculating placeholder height & width could be an expensive operation.
+   * It's a trade off- cost of replicating the element vs. cost of calculating placeholder height & width.
+   */
+  notUsePlaceHolder: boolean;
+
+  /**
+   * If true, arranges Sticky component(s) based on Sticky's 'order' prop in ascending order.
+   */
+  arrangeStickiesBasedOnOrder: boolean;
+
+  /**
+   * Determies when sticky behavior kicks in for Sticky component(s).
+   * There are some calculations which determine if a Sticky component is sticky or non-sticky.
+   * These calculations can be expensive and affect page load time.
+   */
+  containerBehavior: StickyContainerBehaviorType;
+}
+
+/**
+ * {@docCategory ScrollablePane}
+ */
+export enum StickyContainerBehaviorType {
+  /**
+   * This is the default behavior. It can affect page load time.
+   */
+  Default = 0,
+
+  /**
+   * Sticky component(s) will become sticky or non-sticky based on scrolling.
+   * The calculation which determine if a Sticky component is sticky or non-sticky,
+   * are done after user interaction (scrolling) and don't affect page load time.
+   */
+  StickyOnScroll = 1,
```

Comment:
Should this be the default and we drop `StickyAlways` completely? What is the benefit in keeping `StickyAlways` behavior?

Reply:
[See this](https://github.com/OfficeDev/office-ui-fabric-react/pull/9525#discussion_r301497664)